src/components/team/InviteMembers.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from "react";
+import Popup from "reactjs-popup";
+import useAxios from "../../common/hooks/useAxios";
+import { isErrorResponse } from "../../interfaces/ErrorResponse";
+import { IPlayer } from "../../interfaces/IPlayer";
+
+// const tempPlayerList = [
+//     {
+//         id: "fff1",
+//         firstName: "Noah",
+//         lastName: "Roerig",
+//         email: "noahr1936@gmail.com",
+//         image: "",
+//     },
+//     {
+//         id: "fff2",
+//         firstName: "Stevan",
+//         lastName: "Perrino",
+//         email: "noahr1936@gmail.com",
+//         image: "",
+//     },
+//     {
+//         id: "fff3",
+//         firstName: "Jimmy",
+//         lastName: "Cannon",
+//         email: "noahr1936@gmail.com",
+//         image: "",
+//     },
+//     {
+//         id: "fff4",
+//         firstName: "Jacob",
+//         lastName: "Hropoff",
+//         email: "noahr1936@gmail.com",
+//         image: "",
+//     },
+// ];
+
+interface IInviteMembers {
+    teamId: number;
+}
+
+function InviteMembers({ teamId }: IInviteMembers) {
+    const [isLocked, setIsLocked] = useState(false);
+    const [option, setOption] = useState<string>("");
+    const [players, setPlayers] = useState<IPlayer[]>();
```

Comment:
why not do `useState<IPlayer[]>([])` instead. This should help with a lot so you don't have to do an extra check to see if the players variable is not undefined.,

Reply:
@ line 50 and 93